mbed_greentea/mbed_test_api.py

```diff
@@ -244,6 +244,7 @@ def get_binary_host_tests_dir(binary_path, level=2):
         # Example:
         # $ mbedhtrun -p :9600 -f "tests-mbed_drivers-generic_tests.bin" -m K64F --grm raas_client:10.2.203.31:8000
         cmd += ['--grm', global_resource_mgr]
+        cmd += ['-P',"300"] #required higher timeout  due to network operations.
```

Comment:
is it generally for every tests? sounds a bit short especially for hw tests. is there another mechanism to configure timeout when test takes longer?